pkg/proxy/ipvs/proxier.go

```diff
@@ -394,19 +401,20 @@ func NewProxier(ipFamily v1.IPFamily,
 		natRules:              proxyutil.NewLineBuffer(),
 		filterChains:          proxyutil.NewLineBuffer(),
 		filterRules:           proxyutil.NewLineBuffer(),
-		netlinkHandle:         NewNetLinkHandle(ipFamily == v1.IPv6Protocol),
+		netlinkHandle:         NewNetLinkHandle(ctx, ipFamily == v1.IPv6Protocol),
 		ipset:                 ipset,
 		nodePortAddresses:     nodePortAddresses,
 		networkInterfacer:     proxyutil.RealNetwork{},
-		gracefuldeleteManager: NewGracefulTerminationManager(ipvs),
+		gracefuldeleteManager: NewGracefulTerminationManager(ctx, ipvs),
+		logger:                logger,
 	}
 	// initialize ipsetList with all sets we needed
 	proxier.ipsetList = make(map[string]*IPSet)
 	for _, is := range ipsetInfo {
-		proxier.ipsetList[is.name] = NewIPSet(ipset, is.name, is.setType, (ipFamily == v1.IPv6Protocol), is.comment)
+		proxier.ipsetList[is.name] = NewIPSet(ctx, ipset, is.name, is.setType, (ipFamily == v1.IPv6Protocol), is.comment)
 	}
 	burstSyncs := 2
-	klog.V(2).InfoS("ipvs sync params", "ipFamily", ipt.Protocol(), "minSyncPeriod", minSyncPeriod, "syncPeriod", syncPeriod, "burstSyncs", burstSyncs)
+	logger.V(2).Info("ipvs sync params", "ipFamily", ipt.Protocol(), "minSyncPeriod", minSyncPeriod, "syncPeriod", syncPeriod, "burstSyncs", burstSyncs)
```

Comment:
Remove "ipFamily" here. It's included already (withValue)

Reply:
Beware that some might run with `ContextualLogging` feature gate turned off. We only enabled it by default last week, and it can still be turned off.

If it's okay to not log `ipFamily` in that case, then it can be removed here.

Reply:
Even with `ContextualLogging=false` the "ipFamily" is logged. Just not twice. So this is OK under any circumstance.
```
{"ts":1711696948833.9749,"caller":"ipvs/proxier.go:417","msg":"ipvs sync params","v":2,"ipFamily":"IPv4","minSyncPeriod":"0s","syncPeriod":"30s","burstSyncs":2}
```